base_layer/wallet/src/storage/database.rs

```diff
@@ -79,6 +80,8 @@ pub enum DbKey {
     WalletBirthday,
     LastAccessedNetwork,
     LastAccessedVersion,
+    BurntProofId(i32),
```

Comment:
i32 is very specific to sqlite. It should be a different type here, and only converted to i32 in the sqlite lib

Reply:
not using DbKeys anymore